src/libraries/Microsoft.PowerFx.Core/Public/Values/VoidValue.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Types
+{
+    public class VoidValue : ValidFormulaValue
+    {
+        internal VoidValue(IRContext irContext) 
+            : base(irContext)
+        {
+        }
+
+        public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettings settings)
+        {
+            sb.Append($"\"Void\"");
```

Comment:
>sb.Append($"\"Void\"") [](http://example.com/codeflow?start=12&length=22)

Will this cause one of the round-trip tests to fail? `"Void"` is of type string, not of type Void.

Reply:
I am just keeping this for now as a place holder, thinking of tackling it at last.

Reply:
yes  - ideally, this produces an expression that evaluates to void. 
However, we do have a flag in settings, because sometimes that expression can be very strange. So we have a "Friendly" flag to produce more human readable forms. 

Blank() is an example - technically is typed, so should be produced as If(false, 0); // blank  of number type.

Reply:
this is updated now.